src/form/input/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,65 @@
+import CheckIcon from "../../../ui/icons/check.svg";
+
+import "./_checkbox.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+export interface CheckboxInputItem {
+  testid: string;
+  id: string;
+  content: React.ReactNode;
+  inputProps: {
+    htmlFor: string;
+    value: string;
+    name: string;
+  };
+}
+
+export interface CheckboxInputProps {
+  item: CheckboxInputItem;
+  onSelect: (name: string, item: CheckboxInputItem) => void;
```

Comment:
```suggestion
  onSelect: (item: CheckboxInputItem, event?: React.SyntheticEvent<HTMLInputElement>) => void;
```

Let's change the order and pass the whole event as the second argument